Mathlib/CategoryTheory/Monoidal/Cartesian/Grp_.lean

```diff
@@ -131,3 +133,86 @@ lemma Grp_Class.inv_comp (f : X ⟶ G) (g : G ⟶ H) [IsMon_Hom g] : f⁻¹ ≫
   simp [Hom.inv_def,IsMon_Hom.inv_hom]
 
 lemma Grp_Class.inv_eq_inv : ι = (𝟙 G)⁻¹ := by simp [Hom.inv_def]
+
+@[reassoc]
+lemma Grp_Class.comp_div (f : X ⟶ Y) (g h : Y ⟶ G) : f ≫ (g / h) = f ≫ g / f ≫ h :=
+  ((yonedaGrp.obj (.mk' G)).map f.op).hom.map_div g h
+
+@[reassoc]
+lemma Grp_Class.div_comp (f g : X ⟶ G) (h : G ⟶ H) [IsMon_Hom h] :
+    (f / g) ≫ h = (f ≫ h) / (g ≫ h) :=
+  ((yonedaGrp.map (Grp_.homMk h)).app (.op X)).hom.map_div f g
+
+@[reassoc]
+lemma Grp_Class.comp_zpow (f : X ⟶ Y) (g : Y ⟶ G) : ∀ n : ℤ, f ≫ g ^ n = (f ≫ g) ^ n
+  | (n : ℕ) => by simp [comp_pow]
+  | .negSucc n => by simp [comp_pow, comp_inv]
+
+lemma Grp_Class.mul_inv_rev [BraidedCategory C] : μ ≫ ι = (ι[G] ⊗ ι) ≫ (β_ _ _).hom ≫ μ := by
+  calc
+    _ = (fst G G * snd G G) ≫ ι := by rw [mul_eq_mul]
+    _ = (snd G G)⁻¹ * (fst G G)⁻¹ := by simp [← Hom.inv_def]
+    _ = lift (snd G G ≫ ι) (fst G G ≫ ι) ≫ μ := rfl
+    _ = lift (fst G G ≫ ι) (snd G G ≫ ι) ≫ (β_ G G).hom ≫ μ := by simp
+    _ = (ι[G] ⊗ ι) ≫ (β_ _ _).hom ≫ μ := by simp
```

Comment:
I am guessing `by simp [mul_eq_mul, inv_eq_inv]` will get you very close to the goal if not close it directly.

Reply:
Apologies I don't see it

Reply:
More generally, we could probably write a tactic that does this. Curate a simpset, do simp [that] + group or abel or aesop_cat or whatever.

Reply:
Yeah, very good point. I'll try collecting more examples